Casks/o/open-data-editor.rb

```diff
@@ -0,0 +1,22 @@
+cask "open-data-editor" do
+  version "1.5.1"
+  sha256 "d211779d1bda0836fb5325e6c81a119d404c9b6e266ebfe62ccdf68c27ff54b5"
+
+  url "https://github.com/okfn/opendataeditor/releases/download/v#{version}/opendataeditor-macos-#{version}.dmg"
```

Comment:
This change will be required if the homepage is updated to my previous suggestion.

```suggestion
  url "https://github.com/okfn/opendataeditor/releases/download/v#{version}/opendataeditor-macos-#{version}.dmg",
       verified: "github.com/okfn/opendataeditor"